src/components/mobile-menu-container/index.tsx

```diff
@@ -68,7 +70,11 @@ const MobileAuthenticationControls = () => {
 		)
 	} else if (showAuthenticatedUI) {
 		content = (
-			<MobileUserDisclosure items={getUserMenuItems({ signOut })} user={user} />
+			<MobileUserDisclosure
+				items={getUserMenuItems({ signOut })}
+				user={user}
+				initialOpen={asPath.includes('/profile') ? true : false}
```

Comment:
Checking for `includes('/profile')` ends up being caught on paths like `/waypoint/docs/runner/profiles`, so what do you think about using `startsWith` instead? Also, `includes` and `startsWith` both return a boolean, so you can simplify this to just pass the return value of the function. 😊

```suggestion
				initialOpen={asPath.startsWith('/profile')}
```

Reply:
Ohhhh good call! Will update 👍 ty